core/src/main/java/kafka/server/share/DelayedShareFetch.java

```diff
@@ -335,13 +336,29 @@ else if (isolationType == FetchIsolation.HIGH_WATERMARK)
 
     }
 
-    private LinkedHashMap<TopicIdPartition, LogReadResult> readFromLog(LinkedHashMap<TopicIdPartition, FetchRequest.PartitionData> topicPartitionData) {
+    private LinkedHashMap<TopicIdPartition, LogReadResult> readFromLog(
+        LinkedHashMap<TopicIdPartition, Long> topicPartitionFetchOffsets,
+        LinkedHashMap<TopicIdPartition, Integer> partitionMaxBytes) {
```

Comment:
nit: just better readable:
```suggestion
        LinkedHashMap<TopicIdPartition, Integer> partitionMaxBytes
    ) {
```

Reply:
I have aligned it like other functions in the code.

Reply:
I have aligned it like other functions in the file.